stack.yaml

```diff
@@ -1,6 +1,6 @@
 # stack yaml for ghc builds
 
-resolver: lts-14.18
+resolver: lts-14.27
```

Comment:
Please don't do this unless there is a clear motivating reason, while in Pact the impact is small, in Chainweb it can result it a rebuild of Pact which slows development time. 

If there is not a clear improvement, "It's good to keep up with LTS's just because" is not a sufficient excuse.

Reply:
Chainweb is already on `14.27`, so there would be no rebuild.

Reply:
Yes I just saw that so this is fine but going forward this can no longer be knee-jerk.